### PR TITLE
docs: add weekly report 0426(en)

### PR DESCRIPTION
**Pull request by Yilialinn**
Fixes: #[Add issue number here]

Changes:

Screenshots of the change:

### CODE REVIEW

File: blog/en/blog/2023/04/26/weekly-report-0426.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+![Apache APISIX New Contributors](https://static.apiseven.com/uploads/2023/04/26/hs1igYPY_%E6%96%B0%E6%99%8B%E8%B4%A1%E7%8C%AE%E8%80%85%E6%B5%B7%E6%8A%A5-%E8%8B%B1%E6%96%8720230426.png)
+
+## Highlights of Recent Features
+
+### Apache APISIX
+
+- [Support bypassing Admin API Auth by configuration](https://github.com/apache/apisix/pull/9147)（Contributor:  [An-DJ](https://github.com/An-DJ)）
+
+- [Support store route's cert in secrets manager](https://github.com/apache/apisix/pull/9247)（Contributor:  [soulbird](https://github.com/soulbird)）
```
Comment (SkyeYoung) [inline blog/en/blog/2023/04/26/weekly-report-0426.md]:
```suggestion
- [Support bypassing Admin API Auth by configuration](https://github.com/apache/apisix/pull/9147)(Contributor: [An-DJ](https://github.com/An-DJ))

- [Support store route's cert in secrets manager](https://github.com/apache/apisix/pull/9247)(Contributor: [soulbird](https://github.com/soulbird))
```

**Review by SkyeYoung — CHANGES_REQUESTED** [review]

**Review by SkyeYoung — APPROVED** [review]

**Comment by guoqqqi** [comment]
By the way, why split it into two PRs to complete it?

#1570

**Review by guoqqqi — APPROVED** [review]

**Comment by Yilialinn** [comment]
> 

1571 English version while 1570 Chinese

**Comment by Yilialinn** [comment]
> By the way, why split it into two PRs to complete it?
> 
> #1570

1571 English version while 1570 Chinese

**Comment by guoqqqi** [comment]
> > By the way, why split it into two PRs to complete it?
> > #1570
> 
> 1571 English version while 1570 Chinese

Yes I know, because I saw the PR was submitted at the same time, I was just curious why it was not done in the same PR.

**Pull request merged**